test/screenshot/screenshot.js

```diff
@@ -200,6 +200,8 @@ export default class Screenshot {
    */
   async takeScreenshot_() {
     const browser = await puppeteer.launch();
+    const version = await browser.version();
+    console.log('VERSION OF BROWSER: ', version);
```

Comment:
oopsies

Reply:
I actually left this in since it verified one less thing that is different from travis. I will remove :)